From 21616a198bee14a9a95a6dacedcd1b1eca175fc6 Mon Sep 17 00:00:00 2001
From: Ani Sinha <anisinha@redhat.com>
Date: Mon, 6 May 2024 11:50:49 +0530
Subject: [PATCH] Disable automatic log collector

RH-Author: Ani Sinha <anisinha@redhat.com>
RH-MergeRequest: 11: Disable automatic log collector
RH-Jira: RHEL-7273
RH-Acked-by: Emanuele Giuseppe Esposito <eesposit@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Commit: [1/1] 7a8002c626b45eb5e90fa0fe82694639d1e5b93d (anisinha/centos-wa-linux-agent)

Log collector is enabled by default. Due to security concerns from our customers
disable it.

Jira: https://issues.redhat.com/browse/RHEL-7273
Upstream: RHEL only.

Signed-off-by: Ani Sinha <anisinha@redhat.com>
---
 config/waagent.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/waagent.conf b/config/waagent.conf
index 14cd01c8..25ee83d3 100644
--- a/config/waagent.conf
+++ b/config/waagent.conf
@@ -73,7 +73,7 @@ Logs.Verbose=n
 # Logs.Console=y
 
 # Enable periodic log collection, default is y
-Logs.Collect=y
+Logs.Collect=n
 
 # How frequently to collect logs, default is each hour
 Logs.CollectPeriod=3600
-- 
2.39.3

